peaks.js.d.ts

```diff
@@ -11,6 +11,7 @@ declare module 'peaks.js' {
     color?: string;
     labelText?: string;
     id?: string;
+    update: (options: Object) => void;
```

Comment:
Using object is proabably too generic. I'd suggest:
```typescript
update: (options: SegmentUpdateValues) => void;
```
and
```typescript
interface SegmentUpdateValues {
  startTime?: number;
  endTime?: number;
  editable?: boolean;
  color?: string;
  labelText?: string;
}
```
(and similar for `Point.update`)

Reply:
done